osquery/dispatcher/scheduler.cpp

```diff
@@ -52,6 +53,11 @@ FLAG(bool,
      false,
      "Log the running scheduled query name at INFO level");
 
+FLAG(bool,
+     schedule_allow_denylisted_events,
```

Comment:
It's a bit of a nitpick, but this name is a mouthful. Also, what should the default be -- it changes the words...

Thinking aloud, `skip` or `enforce` let's us avoid the doubling of `allow` and `deny`. Something like:
* `skip_events_in_denylist`
* `enforce_denylist_for_events`

Reply:
I don't have strong feelings either way. I am tempted to make this `HIDDEN` because I cannot see someone wanting to disable it.